index.html

```diff
@@ -248,5 +249,6 @@ <h2 class="success__title">Изображение успешно загруже
   </template>
   <script src="/js/main.js" type="module"></script>
   <script src="pristine/pristine.min.js"></script>
+  <script src="/nouislider/nouislider.js"></script>
```

Comment:
```suggestion
  <script src="pristine/pristine.min.js"></script>
  <script src="/nouislider/nouislider.js"></script>
    <script src="/js/main.js" type="module"></script>
```
сначала должны быть библиотеки, так как ты используешь их код дальше